infrastructure/gcp/keep-test/google-functions/keep-faucet/parse-account-address.js

```diff
@@ -3,18 +3,18 @@ const url = require('url')
 
 exports.parseAccountAddress = (request, response) => {
   requestUrl = url.parse(request.url, true)
-  if (requestUrl.query.address) {
-    return (address = requestUrl.query.address)
-  } else if (!/^(0x)?[0-9a-f]{40}$/i.test(address)) {
-    // check if it has the basic requirements of an address
+  if (requestUrl.query.account) {
+    return (account = requestUrl.query.account)
+  } else if (!/^(0x)?[0-9a-f]{40}$/i.test(account)) {
```

Comment:
`account` is never defined here. Mayhaps after we declare `requestUrl` we can `const account = requestUrl.query.account` and then use `account` throughout the rest of the function.

Reply:
oh my^2